lib/rubocop/rspec/cop_helper.rb

```diff
@@ -49,7 +49,7 @@ def _investigate(cop, processed_source)
     team = RuboCop::Cop::Team.new([cop], nil, raise_error: true)
     report = team.investigate(processed_source)
     @last_corrector = report.correctors.first || RuboCop::Cop::Corrector.new(processed_source)
-    report.offenses
+    report.offenses.reject(&:disabled?)
```

Comment:
In order to test this, I needed to update `expect_offense` and `expect_no_offenses` to not consider disabled offenses. There was only one test (in `spec/rubocop/cop/layout/line_length_spec.rb`) that was affected by this change.